linkml/generators/docgen.py

```diff
@@ -159,6 +158,7 @@ def __post_init__(self):
         if self.example_directory:
             self.example_runner = ExampleRunner(input_directory=Path(self.example_directory))
         super().__post_init__()
+        self.schemaview = SchemaView(self.schema, merge_imports=self.mergeimports)
```

Comment:
I see `self.mergeimports=False` here but https://github.com/linkml/linkml/issues/1282 is not fixed.

Reply:
Wait, let me check something.

Reply:
I think you might be right @noelmcloughlin, we might have to pass the value of `self.mergeimports` to `all_classes()`, `all_slots()`, etc. in docgen.py like you have in PR #1283

Reply:
I will check. I'm having an inconvenient bug in my parent schema ;-( just trying to fix that and I can test for sure.

Reply:
Awesome, thank you. I've updated this PR.